scripts/generate-authors.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ts-node
```

Comment:
We can drop this if those files are not ts, right?

Reply:
👍